ban/tests/auth/test_models.py

```diff
@@ -12,30 +10,3 @@ def test_user_password_is_hashed():
     user = models.User.get(models.User.id == user.id)
     assert user.password != password
```

Comment:
No big deal here but I would also comfort checking the length of the string (32? depending on the hash)